examples/rally-api/src/integration-test/groovy/yakworks/rest/RestApiListMaxSpec.groovy

```diff
@@ -0,0 +1,73 @@
+package yakworks.rest
+
+import grails.gorm.transactions.Rollback
+import grails.testing.mixin.integration.Integration
+import org.springframework.http.ResponseEntity
+import spock.lang.Specification
+import yakworks.rally.orgs.model.Org
+import yakworks.rest.client.OkAuth
+import yakworks.rest.client.WebClientTrait
+
+@Integration
+class RestApiListMaxSpec extends Specification  implements WebClientTrait {
```

Comment:
can we test what it looks like when it goes past the timeout?

Reply:
Test added